Src/library/cURL/curl_easy_externals.e

```diff
@@ -17,14 +17,12 @@ feature -- Command
 	init: POINTER
 			-- Declared as curl_easy_init().
 		require
-			dynamic_library_exists: is_dynamic_library_exists
+			dynamic_library_exists: is_dynamic_library_exists or is_static
 		local
 			l_api: POINTER
 		do
 			l_api := api_loader.api_pointer ("curl_easy_init")
-			if l_api /= default_pointer then
-				Result := c_init (l_api)
-			end
+			Result := c_init (l_api)
```

Comment:
What if l_api is default pointer and dynamic lib is the default ?

Reply:
the code violates the precondition of c_init ..

same comment for all code interfacing with the following cURL C externals

Reply:
you are right, that's why in the previous commit  I've done something like this

l_api := api_loader.api_pointer ("curl_easy_init")
if l_api /= default_pointer then
	Result := c_init (l_api)
else
	Result := c_init (default_pointer)
end	

But, we can check precondition so we will be safe, so we can write something like


l_api := api_loader.api_pointer ("curl_easy_init")
if l_api /= default_pointer  or is_static then
	Result := c_init (l_api)
end	
What do you think?